tools/cjxl_ng_main.cc

```diff
@@ -46,35 +56,19 @@ DEFINE_bool(responsive, false, "[modular encoding] do Squeeze transform");
 
 DEFINE_bool(progressive, false, "Enable progressive/responsive decoding.");
 
-DEFINE_bool(progressive_ac, false,  // TODO(tfish): Wire this up.
-            "Use progressive mode for AC.");
+DEFINE_bool(progressive_ac, false, "Use progressive mode for AC.");
 
-DEFINE_bool(qprogressive_ac,
-            false,  // TODO(tfish): Wire this up.
-                    // TODO(tfish): Clarify what this flag is about.
-            "Use progressive mode for AC.");
+DEFINE_bool(qprogressive_ac, false, "Use progressive mode for AC.");
 
-DEFINE_bool(use_experimental_encoder_heuristics,
-            false,  // TODO(tfish): Wire this up.
-            "Use new and not yet ready encoder heuristics");
+DEFINE_bool(modular_lossy_palette, false, "Use delta-palette.");
 
 DEFINE_bool(jpeg_transcode, false,  // TODO(tfish): Wire this up.
             "Do lossy transcode of input JPEG file (decode to "
             "pixels instead of doing lossless transcode).");
 
-DEFINE_bool(jpeg_transcode_disable_cfl, false,  // TODO(tfish): Wire this up.
-            "Disable CFL for lossless JPEG recompression");
-
 DEFINE_bool(premultiply, false,  // TODO(tfish): Wire this up.
             "Force premultiplied (associated) alpha.");
 
-DEFINE_bool(centerfirst, false,  // TODO(tfish): Wire this up.
-            "Put center groups first in the compressed file.");
```

Comment:
this is removed, because it is the same as the "group_order" flag, correct?

Reply:
Yes, I think this functionality is now available via --group_order.